tests/src/test/java/alluxio/worker/AlluxioWorkerRestApiTest.java

```diff
@@ -91,6 +97,37 @@ public void getCapacityBytesTest() throws Exception {
         NO_PARAMS, HttpMethod.GET, capacityBytes).run();
   }
 
+  private Configuration mockConfiguration() {
```

Comment:
@aaudiber your suggesting will not work because the config will always contain some default values